.github/workflows/nightly-ci.yml

```
@@ -476,6 +454,17 @@ jobs:
       pytest_markers: 'vllm and xpu_1'
       source_ref: ${{ needs.resolve-source-sha.outputs.source_sha }}
 
+  vllm-deploy-xpu:
+    name: vllm-xpu-deploy
+    needs: [resolve-source-sha, compute-release-mode]
+    if: ${{ needs.compute-release-mode.outputs.run_tests == 'true' }}
+    uses: ./.github/workflows/xpu-deploy.yaml
+    with:
+      framework: vllm
+      source_ref: ${{ needs.resolve-source-sha.outputs.source_sha }}
+      image_tag_suffix: '-nightly'
```

actionlint (1.7.12)

[error] input "image_tag_suffix" is not defined in "./.github/workflows/xpu-deploy.yaml" reusable workflow. defined inputs are "framework", "registry_host", "source_ref"

(workflow-call)